user-guide/Standard_Apps/EPM/EPM_I-DOCSIS/I-DOCSIS_Create_CCAP_CM_pair.md

```diff
@@ -0,0 +1,72 @@
+---
+uid: I-DOCSIS_Create_CCAP_CM_pair
+---
+
+# Creating a new CCAP/CM pair
+
+To create a new CCAP/CM pair:
+
+1. In DataMiner Cube, go to apps > Automation.
+
+1. In the pane on the left, select the script *EPM_I_DOCSIS_AddNewCcapCmPair*.
+
+1. In the lower right corner, click *Execute*.
+
+1. UI steps:
```

Comment:
Are these the different steps in the wizard? If yes, I'd recommend changing this as follows (and changing the steps below in a similar manner):

```
1. In the lower right corner, click *Execute*.

   This will open a wizard with several steps.

1. In the *General* step:

   1. Define the element name. 

      ...
```

Reply:
Yup this is how you execute the automation scrip. After clicking on the "Execute" button a UI is going to appear and then is when the steps come up